src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -3837,6 +3837,63 @@ public LazyTryStatement(Lazy<IBlockStatement> body, Lazy<ImmutableArray<ICatchCl
     }
 
     /// <summary>
```

Comment:
Out of curiosity, how is this file generated? What are the inputs used and the command to run?